rotkehlchen/chain/optimism/modules/velodrome/decoder.py

```diff
@@ -36,11 +64,224 @@ def __init__(
         )
         self.pools, self.gauges = read_velodrome_pools_and_gauges_from_cache()
 
-    def _decode_velodrome_events(self) -> None:
-        pass
+    def _decode_add_liquidity_events(
+            self,
+            tx_log: 'EvmTxReceiptLog',
+            decoded_events: list['EvmEvent'],
+    ) -> DecodingOutput:
+        """Decodes events that add liquidity to a velodrome pool"""
```

Comment:
state the errors that it can raise and also the version of velodrome that this is compatible with